src/Features/SupportPageComponents/HandlesPageComponents.php

```diff
@@ -10,14 +12,18 @@ function __invoke()
         // This way, users can pass Livewire components into Routes as if they were
         // simple invokable controllers. Ex: Route::get('...', SomeLivewireComponent::class);
         $html = null;
+        $layout = null;
 
         $layoutConfig = SupportPageComponents::interceptTheRenderOfTheComponentAndRetreiveTheLayoutConfiguration(function () use (&$html) {
             $params = SupportPageComponents::gatherMountMethodParamsFromRouteParameters($this);
 
             $html = app('livewire')->mount($this::class, $params);
         });
 
-        $layoutConfig = $layoutConfig ?: new PageComponentConfig;
+        $reflectionClass = new \ReflectionClass($this);
+        $layout = collect($reflectionClass->getAttributes(Layout::class))->first()?->getArguments()[0];
```

Comment:
`collect` function takes extra resources to extract array and create it's own, so I think it is better to just use array index [0] without adding `collect()` just for ->first() methods, that equivalent to array[0]

Reply:
Undersootd! Fixed on bc930f96deaa3a8fbf860575ef860c4be383f171 as well.

Reply:
I implemented as array with `isset`, but not too sure we need to call it. Please let me know if we don't need to check array indices with `isset`.

Reply:
We could just use ->getAttributes()[0] ?? null 
Without any additional if statements :)

Reply:
Got it! I fixed it on ce10eac16d0683eafac50c48225b651006a08a7e.

I mimicked below code:
https://github.com/livewire/livewire/blob/main/src/Features/SupportLazyLoading/SupportLazyLoading.php#L35

Additionaly, put null check for `$layoutAttribute` before calling `->getArguments()[0]` as we would get null if we don't have any `Layout` attribute.

Please let me know if there's anything i can improve.

Reply:
Looks good. Did you manually test it? In my case I see empty div after page load and nothing changes 😩

Reply:
Thank you for your review. Hmm, I manually test it and for me it seems works.

Reply:
@DanielSpravtsev is there anything I can help with?

Reply:
@gazzoy Nope. I think we can call @joshhanley for test and approval if everything works as expected.